packages/api-genesis/test/genesis.query.test.ts

```diff
@@ -4,37 +4,50 @@ import util from '@cardano-graphql/util'
 import { TestClient } from '@cardano-graphql/util-dev'
 import { buildClient } from './util'
 
-const shelleyTestnetGenesis = require('../../../config/network/shelley_testnet/genesis_shelley.json')
-const mainnetCandidateGenesis = require('../../../config/network/mainnet_candidate/genesis_shelley.json')
+const genesisFiles = {
```

Comment:
Could these be moved to a local test_files folder, rather than imported like they are?

Reply:
To tie up this loose end....
These are paths to genesis files I will support, and load into the containers dynamically at runtime, so they're not really "test files".